File: client/client_desktop_app/model/order_status.py
from dataclasses import dataclass

from dataclasses_json import dataclass_json, LetterCase


@dataclass_json(letter_case=LetterCase.CAMEL)
@dataclass
class OrderStatus:
    asset_name: str
    quantity: int
    order_type: str
    order_subtype: str
    order_price: float
    client_id: str
    broker_id: str
    id: str
    status: str

File: stock/stock_exchange_core/domain/model/order_status.go
package model

type OrderStatusEnum string

const (
	CREATED   OrderStatusEnum = "CREATED"
	FULFILLED                 = "FULFILLED"
)

type OrderStatus struct {
	AssetName      string          `json:"assetName"`
	Quantity       int             `json:"quantity"`
	OrderType      string          `json:"orderType"`
	OrderSubtype   string          `json:"orderSubtype"`
	OrderPrice     float64         `json:"orderPrice"`
	ExecutionPrice float64         `json:"executionPrice"`
	ClientId       string          `json:"clientId"`
	BrokerId       string          `json:"brokerId"`
	Id             string          `json:"id"`
	Status         OrderStatusEnum `json:"status"`
}
